README.md

```
@@ -110,12 +114,13 @@
 - [Sets](#sets)
 - [Intersections](#intersections)
 - [Recursive types](#recursive-types)
-  - [ZodType with ZodEffects](#zodtype-with-zodeffects)
+  - [ZodType with ZodTransform](#zodtype-with-ZodTransform)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
Expected: #zodtype-with-zodtransform; Actual: #zodtype-with-ZodTransform

(MD051, link-fragments)